File: ontrack-ui-support/src/main/java/net/nemerosa/ontrack/ui/support/IDToString.java
package net.nemerosa.ontrack.ui.support;

import net.nemerosa.ontrack.model.structure.ID;
import org.springframework.core.convert.converter.Converter;

public class IDToString implements Converter<ID, String> {
    @Override
    public String convert(ID source) {
        return source.toString();
    }
}

File: ontrack-ui-support/src/main/java/net/nemerosa/ontrack/ui/support/IDToString.kt
package net.nemerosa.ontrack.ui.support

import net.nemerosa.ontrack.model.structure.ID
import org.springframework.core.convert.converter.Converter

class IDToString : Converter<ID, String> {
    override fun convert(source: ID): String {
        return source.toString()
    }
}